src/ripple/server/impl/ServerHandlerImp.cpp

```diff
@@ -521,7 +520,7 @@ struct ParsedPort
 
     boost::optional<boost::asio::ip::address> ip;
     boost::optional<std::uint16_t> port;
-    boost::optional<bool> allow_admin;
+    boost::optional<std::vector<beast::IP::Address>> admin_ip;
```

Comment:
@nbougalis The comma separated syntax is not unique to this config element. The protocol is specified exactly the same way.